plugins/quota/rest.go

```diff
@@ -13,8 +13,33 @@ import (
 	"github.com/gluster/glusterd2/pkg/errors"
 	"github.com/gorilla/mux"
 	"github.com/pborman/uuid"
+	log "github.com/sirupsen/logrus"
 )
 
+// IsQuotaAffectedNode returns true if the volume exists on that node and has quota enabled
+func IsQuotaAffectedNode() bool {
+	volumes, e := volume.GetVolumes()
+	if e != nil {
+		log.WithError(e).Error("Failed to get volumes")
+		return true
+	}
+	for _, v := range volumes {
+		val, exists := v.Options[volume.VkeyFeaturesQuota]
+		if exists && val == "off" {
```

Comment:
Incomplete condition: `if !exists` is not considered. 

    if (exists && val == "off") || !exists {
        continue
    }